### PR TITLE
fix Listening on port undefined

### PR DESCRIPTION
**Pull request by desirekaleba**
this fixes the message Listening on port undefined logged to console when the server is running but process.env.PORT is undefined.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ app.route('/').get((req, res) => {
   res.render('Load your view here');
 });
 
-app.listen(process.env.PORT || 3000, () => {
-  console.log('Listening on port ' + process.env.PORT);
-});
+const PORT = process.env.PORT || 3000;
+app.listen(PORT, () => {
+  console.log('Listening on port ' + PORT);
+});
```
Comment (naomi-lgbt) [inline server.js]:
```suggestion
});

```
Could we add the EOF new line?

Otherwise, this looks good to me.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (desirekaleba) [reply server.js]: Great. Let me work on it.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me. Thanks for resolving that.

**Comment by desirekaleba** [comment]
Thanks for approving that.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
I approved this PR.  We have done the same as the contributor suggested on most of the boilerplates.  Another approach is to do something like the following:
```js
const listener = app.listen(process.env.PORT || 3000, () => {
  console.log('Listening on port ' + listener.address().port);
});
```

**Comment by RandellDawson** [comment]
@desirekaleba Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Comment by desirekaleba** [comment]
@RandellDawson thank you for merging my suggested changes, I hope for more future contributions.

**Pull request merged**